app/models/incoming_email.rb

```diff
@@ -40,6 +40,6 @@ def datetime
   end
 
   def author
-    from
+    client.preferred_name
   end
```

Comment:
I think these changes are a little risky. If a client's contact info changes, or we have to resolve contact info between clients, this could give the false impression that a message was sent in the past by the current client in the present.
Zendesk did this with requesters, and it made it hard for us to debug when we ran into edge cases.
In other words, I don't think we should attribute messages to some piece of info that is mutable and we expect might change over time. If it just displays the "from" info that it pulled from the message itself, we prevent the risk of misattribution if we ever have to resolve weird edge cases involving contact info.